src/ui/server/constants/routes/insurance/index.ts

```diff
@@ -26,6 +26,7 @@ export const INSURANCE_ROUTES = {
     NEED_TO_START_AGAIN: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/need-to-start-again`,
   },
   APPLY_OFFLINE: `${INSURANCE_ROOT}/apply-using-our-form`,
+  YOUR_BUYER_DETAILS: `${INSURANCE_ROOT}/your-buyer-details`,
```

Comment:
Can we extract this into it's own object/file like `EXPORTER_BUSINESS` and `POLICY_AND_EXPORTS` (line 33/34) please? As the buyer section grows they'll be more "your buyer" routes